### PR TITLE
[Backport release-3_4] [Server] WMTS - use resolution for bbox calculation 

### PR DESCRIPTION
**Pull request by rldhont**
Manually backport #9856

### CODE REVIEW

File: src/server/services/wmts/qgswmtsutils.cpp [hunk 1]
```diff
@@ -152,8 +153,7 @@ namespace QgsWmts
     QgsUnitTypes::DistanceUnit unit = tmi.unit;
 
     // constant
-    double unit_to_m = QgsUnitTypes::fromUnitToUnitFactor( tmi.unit, QgsUnitTypes::DistanceMeters );
-    double resolution = points_to_m * scaleDenominator / unit_to_m;
+    double resolution = tmi.resolution;
```
Comment (elpaso) [inline src/server/services/wmts/qgswmtsutils.cpp]: if it's really constant
```suggestion
    const double resolution = tmi.resolution;
```

**Review by elpaso — APPROVED** [review]

File: src/server/services/wmts/qgswmtsutils.cpp (hunk 1 above)
Comment (rldhont) [reply src/server/services/wmts/qgswmtsutils.cpp]: not constant some line after: `resolution = res / 2;`

**Comment by rldhont** [comment]
thanks @elpaso for the review.

File: src/server/services/wmts/qgswmtsutils.cpp (hunk 1 above)
Comment (elpaso) [reply src/server/services/wmts/qgswmtsutils.cpp]: Then better remove the comment ...

**Pull request merged**